tests/api_test.py

```diff
@@ -1919,6 +1919,20 @@ def f(x1, x2, g):
     ):
       jax.vmap(f, (0, 0, None))(jnp.ones(2), jnp.ones(3), jnp.add)
 
+  def test_vmap_inconsistent_sizes_constructs_proper_error_message_kwargs(self):
+    def f(x1, x2, a3):
+      return jnp.add(x1, x2, a3)
```

Comment:
Is it intentional that this is an invalid operation? `add` only takes two arguments. Maybe `return x1 + x2 + a3` would be better?

Reply:
Not intentional. Fixed.